pkg/model/serializer.go

```diff
@@ -47,6 +47,38 @@ func (e EnvVar) MarshalYAML() (interface{}, error) {
 	return e.Name + "=" + e.Value, nil
 }
 
+// UnmarshalYAML Implements the Unmarshaler interface of the yaml pkg.
+func (c *Command) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var multi []string
+	err := unmarshal(&multi)
+	if err != nil {
+		var single string
+		err := unmarshal(&single)
+		if err != nil {
+			return err
+		}
+		if strings.Contains(single, " ") {
```

Comment:
What is this check for? I think doing `strings.Split` would work better here.

Reply:
I don't want to split, it is just a checking to know if it is a single command or not